src/components/AppView/AppBar.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import styled, { css } from 'styled-components'
-import { Transition, animated } from 'react-spring/renderprops'
+import { Transition, animated } from '@react-spring/web'
+// import { Transition, animated } from 'react-spring/renderprops'
```

Comment:
Maybe clean it up?

Reply:
hahaha yes, I will.

I will also clean the dependencies. That will be done in the other PR on #29